test/benchee/benchmark/runner_test.exs

```diff
@@ -146,7 +146,7 @@ defmodule Benchee.Benchmark.RunnerTest do
       assert [memory_consumption] = Enum.uniq(memory_usages)
       assert memory_consumption >= 1
       # depending on the number iterations determined, there can be spikes/changes
-      assert memory_consumption <= 100
+      assert memory_consumption <= 1_000
```

Comment:
woopsie I added a 0 here! ;)

Before this was run in the repeated code with a `num_iterations` of 100 or 10. It seems that we have some small basic overhead that now shows up again more. We can have a look at removing this some time later.

Reply:
Just wrote a small script and memory usage for an empty function without anything seems to be 616 Byte on my machine.